components/shared/ParseHTML.tsx

```diff
@@ -25,15 +25,22 @@ import "prismjs/components/prism-sql";
 import "prismjs/components/prism-mongodb";
 import "prismjs/plugins/line-numbers/prism-line-numbers.js";
 import "prismjs/plugins/line-numbers/prism-line-numbers.css";
-
 interface Props {
   data: string;
 }
 
 export default function ParseHTML({ data }: Props) {
+  const [copied, setCopied] = useState(false);
```

Comment:
In Versel It gives Error Fix it You have defined the copied state but where are you passing this? 


```
<html>
<body>
<!--StartFragment-->
Error: 'copied' is assigned a value but never used.  no-unused-vars
--
21:17:13.324 | 34:9  Error: 'handleCopy' is assigned a value but never used.  no-unused-vars

<!--EndFragment-->
</body>
</html>
```

Reply:
fix the deplyoment issue :